0000-dasherized-test-name-generator-option.md

```diff
@@ -0,0 +1,92 @@
+- Start Date: 2017-03-20
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+(1) Add an option to the test generating commands to allow for the test name to be dasherized instead of being humanized (as it is [at the moment](https://github.com/ember-cli/ember-cli-test-info/blob/master/index.js#L27)), for instance `--dasherize`
+
+OR
+
+(2) Dasherize the test name by default and provide an option to humanize it, to get today's output, for instance `--humanize`
+
+Test name examples for `some-multi-word-service-test.js` :
+
+* With `--dasherize`: `Unit | Service | some-multi-word-service`
+* With `--humanize` (default today): `Unit | Service | some multi word service`
```

Comment:
IMHO this RFC should just set the new default to `dasherize`, but without adding any new options that would increase the complexity